Firestore/Source/Local/FSTLevelDBMigrations.h

```diff
@@ -29,12 +30,13 @@ typedef int32_t FSTLevelDBSchemaVersion;
 /**
  * Returns the current version of the schema for the given database
  */
-+ (FSTLevelDBSchemaVersion)schemaVersionForDB:(std::shared_ptr<leveldb::DB>)db;
++ (FSTLevelDBSchemaVersion)schemaVersion:
+    (firebase::firestore::local::LevelDbTransaction *)transaction;
 
 /**
  * Runs any migrations needed to bring the given database up to the current schema version
  */
-+ (void)runMigrationsOnDB:(std::shared_ptr<leveldb::DB>)db;
++ (void)runMigrations:(firebase::firestore::local::LevelDbTransaction *)transaction;
```

Comment:
`runMigrationsWithTransaction` would be more ObjC-style conforming.

Reply:
Done.